_posts/2020-09-28-intro-developer-advocate.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "Hello I'm Brian, Presto Developer Advocate"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Hello, Presto nation!
+
+I wanted to take a moment to introduce myself! My name is Brian and I’m a new
+developer advocate working at Starburst Data. In this blog, I'll give you a
```

Comment:
Typically only use Starburst in outward facing stuff unless legally required to use full name

Reply:
done